### PR TITLE
feat: add ja

### PR DESCRIPTION
**Pull request by masayukiiizuka**
**Summary**
Add new translations for ja (Japanese)

**Testing**
Made sure Japanese text was translated as expected

### CODE REVIEW

File: package.nls.ja.json [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  "notebooks.SAS.logRenderer": "SAS ログレンダラー",
+  "notebooks.SAS.sasNotebook": "SAS ノートブック",
+  "themes.SAS.dark": "SAS Dark",
+  "themes.SAS.highContrast": "SAS High Contrast",
+  "themes.SAS.light": "SAS Light",
+  "views.SAS.explorer": "エクスプローラー",
+  "views.SAS.libraries": "ライブラリ",
+  "views.SAS.signIn": "サインイン",
+  "views.SAS.unsupportedConnection": "この接続では、SAS Extension for Visual Studio Code 内の SAS コンテンツナビゲーションがサポートされていません。ファイルにアクセスできるのは、エクスプローラービューのみです。",
+  "views.SAS.welcome": "SAS Extension for Visual Studio Code を使用するには、SAS にサインインする必要があります。\n [Sign In](コマンド: SAS.authorize)"
```
Comment (scnwwu) [inline package.nls.ja.json:70]: Hi, please do NOT translate `command:` as it's part of the hyperlink syntax to trigger the command. Thanks.

**Comment by scnwwu** [comment]
Please run `npm run format` to have the files automatically formatted. Thanks.

**Comment by masayukiiizuka** [comment]
revert "command:" and run npm run format.

**Comment by scnwwu** [comment]
Hi, the build still failed. Please make sure to commit LF line ending to satisfy the format check. Please let me know if you'd like help. Thank you.

**Comment by masayukiiizuka** [comment]
I upload files again via webui. Please check.

**Review by scnwwu — APPROVED** [review]

**Comment by scnwwu** [comment]
Great contribution. Thank you!

**Pull request merged**